cypress/e2e/pages/load_safe.pages.js

```diff
@@ -52,13 +55,23 @@ export function verifyIncorrectAddressErrorMessage() {
   cy.get(addressInput).parent().prev('label').contains(invalidAddressFormatErrorMsg)
 }
 
+export function verifyNameLengthErrorMessage() {
+  cy.get(nameInput).parent().prev('label').contains(invalidAddressNameLengthErrorMsg)
+}
+
+export function ScanQRCode(image) {
```

Comment:
Please move locators to top of the document to follow established practise and reuse them in the code.